common/is/clean-database-postgres.sql

```diff
@@ -0,0 +1,11 @@
+-- add tables that need to be truncated
+TRUNCATE TABLE IDN_AUTH_SESSION_STORE CASCADE;
+TRUNCATE TABLE IDN_AUTH_TEMP_SESSION_STORE CASCADE;
```

Comment:
Can you change the existing mysql script also to the same naming convention?

Reply:
Additionally let's add this inside cleanup script directory inside this is directory